From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 6 Jul 2010 13:44:58 -0400
Subject: [i2c] fix exports types for recently added i2c symbols
Message-id: <20100706133528.16794.832.sendpatchset@prarit.bos.redhat.com>
Patchwork-id: 26715
O-Subject: [RHEL5 PATCH BZ 611774]: Fix exports for i2c_new_dummy() and
	i2c_unregister_device()
Bugzilla: 611774
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>

linux-2.6 has i2c_new_dummy() and i2c_unregister_device() as EXPORT_SYMBOL_GPL.
Obviously, these should be _GPL in RHEL5.

Resolves BZ 611774.

P.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/i2c/i2c-core.c b/drivers/i2c/i2c-core.c
index 3cf0b73..591eebd 100644
--- a/drivers/i2c/i2c-core.c
+++ b/drivers/i2c/i2c-core.c
@@ -1185,8 +1185,8 @@ void i2c_unregister_device(struct i2c_client *c)
 {
 	kfree(c);
 }
-EXPORT_SYMBOL(i2c_new_dummy);
-EXPORT_SYMBOL(i2c_unregister_device);
+EXPORT_SYMBOL_GPL(i2c_new_dummy);
+EXPORT_SYMBOL_GPL(i2c_unregister_device);
 
 /* Next four are needed by i2c-isa */
 EXPORT_SYMBOL_GPL(i2c_adapter_dev_release);
